packages/@ember/-internals/runtime/lib/mixins/array.js

```diff
@@ -678,7 +678,18 @@ const ArrayMixin = Mixin.create(Enumerable, {
   },
 
   /**
-    Alias for `mapBy`
+    Alias for `mapBy`, returns the value of the named
+    property on all items in the enumeration.
+
+    ```javascript
+    let people = [{name: 'Joe'}, {name: 'Matt'}];
+
+    people.getEach('name');
+    // ['Joe', 'Matt'];
+
+    people.getEach('unknownProperty');
```

Comment:
`unknownProperty` is a special value for EmberObjects, let's use a different name to avoid confusion:

```suggestion
    people.getEach('nonexistentProperty');
```